src/Common/src/Microsoft/Win32/SafeHandles/SafeEvpCipherCtxHandle.Unix.cs

```diff
@@ -37,11 +37,11 @@ internal static SafeEvpCipherCtxHandle Create()
 
             try
             {
-                memPtr = Marshal.AllocHGlobal(Interop.libcrypto.EVP_CIPHER_CTX_SIZE);
+                memPtr = Marshal.AllocHGlobal(Interop.Crypto.EVP_CIPHER_CTX_SIZE);
```

Comment:
I guess you sort of did already, by making GetEvpCipherCtxSize.  I think I'm okay with that...

Should still tag the issue as fixed, though :)